components/sendinblue/actions/send-transactional-email/send-transactional-email.mjs

```diff
@@ -108,6 +108,14 @@ export default {
 
     return props;
   },
+  methods: {
+    formatEmailProp(prop) {
+      if (typeof prop[0] === "string") {
+        return Object.keys(prop).map((key) => JSON.parse(prop[key]));
+      }
+      return prop;
+    },
```

Comment:
Optional if you want to improve a bit more
```suggestion
function formatEmailProp(prop) {
  if (!Array.isArray(prop)) {
    throw new Error("Field should be an array of strings", prop);
  }
  return Object.values(prop)
    .reduce((reduction, value) => {
      try {
        if (typeof(value) !== "string") {
          throw "Not the right type";
        }
        return reduction.concat(JSON.parse(value));
      } catch (error) {
        console.log(`Error parsing value "${value}"`);
        return reduction;
      }
    }, []);
}
```

Or this version would be much simpler
```suggestion
function formatEmailProp(prop) {
  if (!Array.isArray(prop)) {
    throw new Error("Field should be an array of strings", prop);
  }
  return Object.values(prop).map((value) => JSON.parse(value));
}
```

Reply:
Thanks. I updated it to check that `prop` is an array and to throw an error if not. The issue for this PR is requesting that the fields allow an array of objects, so I don't want to force them to be an array of strings. We just don't want to try and parse the values if they're already objects.